keras/backend/cntk_backend.py

```diff
@@ -2321,9 +2375,8 @@ def get_num_dynamic_axis(x):
 def _reduce_on_axis(x, axis, reduce_fun_name):
     if isinstance(axis, list):
         for a in axis:
-            if isinstance(a, C.Axis) \
```

Comment:
This part wasn't more than 85 characters, you don't need to change it.

Reply:
This change has been roll backed.